addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -555,6 +568,15 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
     _allowParentsEditors($snippet) {
         return this._super(...arguments) && !$snippet[0].classList.contains("s_popup");
     },
+    /**
+     * @override
+     */
+    _allowInTranslationMode($snippet) {
+        return (
+            this._super(...arguments) ||
+            weSnippetEditor.globalSelector.is($snippet, { onlyTextOptions: true })
+        );
+    },
```

Comment:
Should that be in website? Why can't it be the original web_editor implementation?

Reply:
As far as I remember, I did it that way because text options are currently added in `website`.... Moved now to `web_editor` :+1: